pkg/webui/console/store/reducers/user.js

```diff
@@ -51,11 +52,10 @@ const user = function(state = defaultState, { type, payload }) {
         ...state,
         rights: payload,
       }
-    case LOGOUT_SUCCESS:
+    case LOGOUT_FAILURE:
```

Comment:
With this change, the user won't be logged out from the console automatically on an unauthenticated error. The are still places where `logoutSuccess` is dispatched, see [L78](https://github.com/TheThingsNetwork/lorawan-stack/blob/feature/1422-global-logout/pkg/webui/console/store/middleware/logics/lib.js#L78) and [L95](https://github.com/TheThingsNetwork/lorawan-stack/blob/feature/1422-global-logout/pkg/webui/console/store/middleware/logics/events.js#L95)